frontend/www/js/omegaup/typeahead.ts

```diff
@@ -100,6 +100,43 @@ function typeahead<T extends { label: string; value: string }>(
     .trigger('change');
 }
 
+export function nominationTypeahead(
+  elem: JQuery<HTMLElement>,
+  cb?: CallbackType<types.NominationListItem>,
+) {
+  if (!cb) {
+    cb = (event: Event, val: types.NominationListItem) =>
+      $(<EventTarget>event.target).val(val.problem.alias);
+  }
+  elem
+    .typeahead<types.NominationListItem>(
+      {
+        minLength: 3,
+        highlight: false,
+      },
+      {
+        source: typeaheadWrapper(
```

Comment:
Aquí podrías mandar llamar la función que acabas de crear:
```suggestion
        source: typeaheadNominationsWrapper(
```